worker/transcribee_worker/whisper_transcribe.py

```diff
@@ -193,3 +202,50 @@ async def transcribe(
 
     for task in pending:
         task.cancel()
+
+
+async def recombine_split_words(
```

Comment:
maybe we want to also recombine sentences in the future? so that paragraph breaks only occur on sentence end symbols like `.,?!`